kubernetes/apps/ai/namespace.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ai
+  labels:
+    pod-security.kubernetes.io/enforce: privileged
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)